src/libsystemd-network/sd-dhcp-lease.c

```diff
@@ -780,6 +843,37 @@ int sd_dhcp_lease_load(sd_dhcp_lease **ret, const char *lease_file) {
                            "ROUTES", &routes,
                            "CLIENTID", &client_id_hex,
                            "VENDOR_SPECIFIC", &vendor_specific_hex,
+                           "OPTION_224", &options[DHCP_OPTION_PRIVATE_0 - DHCP_OPTION_PRIVATE_START],
```

Comment:
I had originally written this as:

```C
DHCP_OPTION_PRIVATE_START = 224,
DHCP_OPTION_PRIVATE_END = 255,

...

"OPTION_224", &options[0],
"OPTION_225", &options[1],
```

But I didn't like that all of those constants had to be kept in sync; the unrolled loop had to match the length of that array. But maybe it's not a big deal. The compiler would give enough warning about a typo.